e2e/integration/menu.e2e.ts

```diff
@@ -59,10 +55,25 @@ describe('Menu', () => {
   it('should deactivate tag when other is activated', () => {
     const petItem = () => cy.contains('[role=menuitem].-depth1', 'pet');
 
-    petItem()
-      .click({ force: true })
-      .should('have.class', 'active');
+    petItem().click({ force: true }).should('have.class', 'active');
     cy.contains('[role=menuitem].-depth1', 'store').click({ force: true });
     petItem().should('not.have.class', 'active');
   });
+
+  it('should be able to open a response object to see more details', () => {
+    cy.contains('h2', 'Find pet by ID')
+      .scrollIntoView()
+      .wait(100)
+      .parent()
+      .find('div h3')
+      .should('have.text', 'Responses')
+      .parent()
+      .find('div:first button')
+      .click()
+      .should('have.attr', 'aria-expanded', 'true')
+      .parent()
+      .find('div h5')
+      .then($h5 => $h5[0].firstChild!.nodeValue!.trim())
```

Comment:
I wanted to get the value of the h5 element.
If I would just do `.should('have.text', 'Response Schema:')` on that h5 it would throw an error. Because the span values would also be in that comparison.